content/en/docs/ops/integrations/kyverno-authz-server/index.md

```diff
@@ -0,0 +1,464 @@
+---
+title: Kyverno Authz Server
+description: How to integrate with Kyverno Authz Server.
+weight: 33
+keywords: [integration,kyverno,policy,authorization]
+owner: istio/wg-environments-maintainers
+test: no
```

Comment:
Have you considered making this testable?  Should be pretty easy.
Check https://github.com/istio/istio.io/tree/master/tests and let me know if you have any issues